packages/data-models/src/lemma.js

```diff
@@ -73,7 +92,7 @@ class Lemma {
     // eslint-disable-next-line prefer-const
     let resultLemma = {
       word: this.word,
-      language: this.languageCode,
+      languageCode: this.language.toCode(),
```

Comment:
note that this change could affect the wordlist.

Reply:
I've checked usages in the wordlist and tried to fix whatever may cause problems, run unit tests, and tested some wordlist cases manually. That definitely might be not enough, but I hope if there are any issues, they will be caught during Monica's full scale test. Tried to do my best not to break anything here.